lib/util/array.ts

```diff
@@ -4,7 +4,7 @@
 import { objectPathAppend } from "./objectPath"
 import { Tracked, tracked } from "./tracked"
 
-export function arrayMap<T>(array: Tracked<T[]>, f: (value: Tracked<T>) => T) {
+export function arrayMap<T>(array: Tracked<T[]>, f: (value: Tracked<T>) => T): T[] {
```

Comment:
>[] [](start = 80, length = 2)

was this causing issues ?

Reply:
No. I just prefer to have explicit function signature for exported functions.